CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
+* cloudbuild: removed the validation function for the values of `machine_type` field on the `google_cloudbuild_trigger` resource ([#15357](https://github.com/hashicorp/terraform-provider-google/pull/15357))
+* compute: add future_limit in quota exceeded error details for compute resources. ([#15346](https://github.com/hashicorp/terraform-provider-google/pull/15346))
+* compute: added `ipv6EndpointType` and `ipVersion` to `google_compute_address` ([#15358](https://github.com/hashicorp/terraform-provider-google/pull/15358))
+* compute: added `local_ssd_recovery_timeout` field to `google_compute_instance` resource ([#15366](https://github.com/hashicorp/terraform-provider-google/pull/15366))
+* compute: added `local_ssd_recovery_timeout` field to `google_compute_instance_template` resource ([#15366](https://github.com/hashicorp/terraform-provider-google/pull/15366))
+* compute: added `network_interface.ipv6_access_config.external_ipv6_prefix_length` to `google_compute_instance` ([#15358](https://github.com/hashicorp/terraform-provider-google/pull/15358))
+* compute: added `network_interface.ipv6_access_config.name` to `google_compute_instance` ([#15358](https://github.com/hashicorp/terraform-provider-google/pull/15358))
+* compute: added a new type `GLOBAL_MANAGED_PROXY` for the filed `purpose` in the resource `google_compute_subnetwork` ([#15345](https://github.com/hashicorp/terraform-provider-google/pull/15345))
+* compute: added field `instance_lifecycle_policy` to `google_compute_instance_group_manager` and `google_compute_region_instance_group_manager` (ga) ([#15322](https://github.com/hashicorp/terraform-provider-google/pull/15322))
+* compute: added protocol type: UNSPECIFIED in `google_compute_backend_service` as per [release note](https://cloud.google.com/load-balancing/docs/release-notes#July_24_2023)
+ ([#15328](https://github.com/hashicorp/terraform-provider-google/pull/15328))
+* compute: made `network_interface.ipv6_access_config.external_ipv6` configurable in `google_compute_instance` ([#15358](https://github.com/hashicorp/terraform-provider-google/pull/15358))
+* container: added `enable_k8s_beta_apis.enabled_apis` field to `google_container_cluster` ([#15320](https://github.com/hashicorp/terraform-provider-google/pull/15320))
+* container: added `node_config.host_maintenance_policy` field to `google_container_cluster` and `google_container_node_pool` ([#15347](https://github.com/hashicorp/terraform-provider-google/pull/15347))
+* container: added `placement_policy.policy_name` field to `google_container_node_pool` resource ([#15367](https://github.com/hashicorp/terraform-provider-google/pull/15367))
+* container: allowed `enabled_private_endpoint` to be settable on creation for PSC-based clusters ([#15361](https://github.com/hashicorp/terraform-provider-google/pull/15361))
+* container: supported master_global_access_config for flexible cluster ([#15369](https://github.com/hashicorp/terraform-provider-google/pull/15369))
```

Comment:
Add ` around field name, what is the resource here?

Reply:
```suggestion
* container: unsuppressed `private_cluster_config ` when `master_global_access_config` is set in `google_container_cluster` ([#15369](https://github.com/hashicorp/terraform-provider-google/pull/15369))
```